lib/widgets/payment_dialogs/processing_payment_dialog.dart

```diff
@@ -85,6 +89,7 @@ class ProcessingPaymentDialogState extends State<ProcessingPaymentDialog>
 
   _payAndClose() {
     final navigator = Navigator.of(context);
+    final texts = getSystemAppLocalizations();
```

Comment:
We can use `context.texts()` here, `getSystemAppLocalizations()` is reserved for places that has no access to `BuildContext`.
```suggestion
    final texts = context.texts();
```

Reply:
Ignore this.